score-client/src/main/java/bio/overture/score/client/command/UploadCommand.java

```diff
@@ -64,9 +67,19 @@ public class UploadCommand extends RepositoryAccessCommand {
    */
   @Autowired
   private ManifestService manifestService;
-  @Autowired
   private UploadService uploader;
 
+  @Autowired
+  ApplicationContext appContext;
+
+  @Autowired
+  private BeanUtil beanUtil;
+
+  @PostConstruct
+  public void initializeStorageProfile() throws Exception{
```

Comment:
Code formatting `Exception {`.

Looks like score does not have a code formatting plugin... the rest of the overture apps do, maybe we can bring that in here... I'll open another ticket to look into this. For reference, we have used coveoos plugin called fmt-maven-plugin, and it looks like this has moved to spotify now:

https://github.com/spotify/fmt-maven-plugin